concordia/language_model/pytorch_gemma_model.py

```diff
@@ -35,6 +35,7 @@ def __init__(
       *,
       measurements: measurements_lib.Measurements | None = None,
       channel: str = language_model.DEFAULT_STATS_CHANNEL,
+      device: str = 'cpu'
```

Comment:
consider: could this be `Literal['cpu', 'gpu']` or somesuch to make it clearer what values are OK?

Reply:
PyTorch allows user to specify the device with custom strings. Ex: "cuda:0" for GPU 1, "cuda:1" for GPU 2, etc. so there isn't really way to wrap this nicely into a literal